docs/reference/cli-reference/cli-ref-restore.md

```diff
@@ -50,6 +50,25 @@ where `<projectPath>` specifies the location of a solution or a `packages.config
 
 Also see [Environment variables](cli-ref-environment-variables.md)
 
+## Lock file options
+With version 5.3.1, `nuget.exe` supports [lock file as an optional feature](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#locking-dependencies). The lock file can be enabled by setting the MSBuild property `RestorePackagesWithLockFile` for your project.
+```
+<PropertyGroup>
+    <!--- ... -->
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+    <!--- ... -->
+</PropertyGroup>
+```
+
+Following are the options applicable for the lock file funtionality:
+| Option |Description|
+|:---  |:--- |
+| `-UseLockFile` |  Opts into the usage of a lock file. | 
+| `-LockedMode` | Enables locked mode for restore. This is useful in CI/CD scenarios where you want repeatable builds.|   
+| `-ForceEvaluate` | This option is useful with packages with floating version defined in the project. By default, NuGet restore will not update the package version automatically upon each restore unless you run restore with this option. |
+| `-LockFilePath` | Defines a custom lock file location for a project. By default, NuGet supports `packages.lock.json` at the root directory. If you have multiple projects in the same directory, NuGet supports project specific lock file `packages.<project_name>.lock.json` |
```

Comment:
It is not clear from this whether `packages.<project_name>.lock.json` is part of the default behaviour or a suggestion of how to use the `-LockFilePath` option.

Reply:
I agree. Documentation about the default file names should be elsewhere, and the docs for `-LockFilePath` should be short and simple.

Additionally, we don't support having multiple project files in a single directory (technically, we don't support having more than one project use a single `MSBuildProjectExtensionsPath` directory, which is the `obj/` directory by default), so I don't think our docs should imply it. Just say that `packages.project.lock.json` is the default filename, and that `packages.<project_name>.lock.json` is tried if it does not exist. But I don't suggest giving justification as to why it exists or scenarios where it would be used.